paddle/fluid/operators/select_op_helper.h

```diff
@@ -37,8 +41,9 @@ inline int GetBranchNumber(const framework::LoDTensor &mask) {
   framework::TensorCopySync(mask, platform::CPUPlace(), cpu_mask.get());
 #else
   PADDLE_THROW(
-      "This version of PaddlePaddle doen NOT support GPU but got GPU tensor "
-      "Mask in SelectOutputOp. Please compile WITH_GPU option");
+      "This version of PaddlePaddle does NOT support GPU but got GPU tensor "
```

Comment:
Should we throw a platform::errors::PreconditionNotMet ?

Reply:
Yes. Thanks, done.